### PR TITLE
[Documentation] Examples part 2

### PR DESCRIPTION
**Pull request by martinmoene**
See issue #1037 

As a long time past between creating part 1 and this little piece, I'm not sure if it's acceptable to work in smaller badges than with part 1 and just review / merge it, or to expand it first.

### CODE REVIEW

**Comment by horenmar** [comment]
I'd prefer if this was expanded to include all of the reporter parts first.

**Comment by martinmoene** [comment]
>I'd prefer if this was expanded to include all of the reporter parts first.

ok, need to get my act together then ;)

**Comment by horenmar** [comment]
@martinmoene Well, I don't think that waiting for larger piece of work worked. 😃  Are you still around to rebase this to something with more reasonable history?

**Comment by martinmoene** [comment]
@horenmar I'm sorry for leaving this unaddressed.

I'll have to refresh my memory of it... and try to come back on in a few days.

**Comment by martinmoene** [comment]
see if this works, then i'll squash...

**Comment by martinmoene** [comment]
@horenmar the request is in pullable form now, a _pr-lite_ so to speak.

File: docs/list-of-examples.md [hunk 1]
```diff
@@ -3,12 +3,15 @@
 
 ## Already available
 
+- Catch main: [Catch-provided main](../examples/000-CatchMain.cpp)
 - Test Case: [Single-file](../examples/010-TestCase.cpp)
-- Test Case: [Multiple-file 1](../examples/020-TestCase-1.cpp), [2](../examples/020-TestCase-2.cpp)
+- Test Case: [Multiple-file 1](../examples/020-TestCase-1.cpp), [2](../examples/020-TestCase-1.cpp)
```
Comment (horenmar) [inline docs/list-of-examples.md]: That doesn't seem right ;-)

Comment (martinmoene) [reply docs/list-of-examples.md]: Good catch, I'd say it is wrong!

**Comment by martinmoene** [comment]
@horenmar I unwronged it.

**Comment by horenmar** [comment]
Thanks

**Pull request merged**